### PR TITLE
cor/fmt in nix

### PR DESCRIPTION
**Pull request by cor**
Already opening this PR in order to test

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -460,6 +460,35 @@
...
+                nixfmt
+                coreutils
+                rust-nightly
+                taplo
+                nodePackages.prettier
+              ];
+
+              text = ''
+                  # .nix 
+                	find . -name "*.nix" -type f -exec nixfmt {} \;
```
Comment (dzmitry-lahoda) [inline flake.nix:476]: nix checked else where afaik. and used xarga i stead of exec. as xargs more composable and no need detals of adhoc exec

**Review by dzmitry-lahoda — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (cor) [reply flake.nix:476]: Oh yeah good point, will convert to xargs

**Review by KaiserKarel — APPROVED** [review]

File: flake.nix (hunk 1 above)
Comment (cor) [reply flake.nix]: fixed

**Comment by cor** [comment]
@mergify requeue

**Pull request merged**